lib/rototiller/task/beaker_task.rb

```diff
@@ -0,0 +1,74 @@
+require 'rake'
+require 'rototiller/task/flags/cli_flags'
+
+module Rototiller
+  module Task
+    class BeakerTask
+
+      # Beaker CLI flags are stored on a module and mixed in
+      include CLIFlags
+      @@cli_flag_names.each do |flag|
+        attr_accessor flag
+      end
+
+      include ::Rake::DSL if defined?(::Rake::DSL)
+      include EnvVar
+
+      def initialize(*args, &task_block)
+        @name          = args.shift
+        @ruby_opts     = nil
+        @verbose       = true
+        @fail_on_error = true
+
+        define(args, &task_block)
+      end
+
+      def self.define_task(*args, &task_block)
+        self.new(*args, &task_block)
+      end
+
```

Comment:
i don't really understand the importance of this.
i'd expect to just do `BeakerTask.new`
similar to how rspec tasks are done: 
http://www.rubydoc.info/github/rspec/rspec-core/RSpec/Core/RakeTask

i also think this portion should be in a beaker-library at somepoint.  but beaker team isn't really ready for that... so we can leave here for now. let's just make it isolated and uncoupled from other classes except maybe a task class.